source/reference/program.txt

```diff
@@ -112,6 +112,15 @@ GridFS
 :program:`mongofiles` provides a command-line interact to a MongoDB
 :term:`GridFS` storage system.
 
+MongoDB Compass
+---------------
+
+.. versionadded:: 3.6
+
+   MongoDB is packaged with an ``install_compass`` script, which
+   is a platform-specific installer for
+   :ref:`MongoDB Compass Community Edition <compass-index>`.
+
```

Comment:
We'll need:

.. toctree::
   :maxdepth: 1

   /reference/program/install_compass

(source/reference/program/install_compass.txt would have:
.. program:: install_compass

   .. versionadded:: 3.6

   Platform-specific installer for :ref:`MongoDB Compass Community Edition <compass-index>`.

   To run, on Linux/iOS:

   .. code-block:: sh

       python install_compass

   To run, on Windows:

(we might have to check on our Windows box tomorrow).

      
We also should blurb in  3.6 rel notes.

And definitely, stop working :D  It's not that urgent.